c-dependencies/js-compute-runtime/builtins/subtle-crypto.cpp

```diff
@@ -0,0 +1,411 @@
+    builtins::CryptoAlgorithmIdentifier::RSA_PSS};
+std::vector<builtins::CryptoAlgorithmIdentifier> supportedVerifyAlgorithms =
+    supportedSignAlgorithms;
+std::vector<builtins::CryptoAlgorithmIdentifier> supportedDigestAlgorithms{
+    builtins::CryptoAlgorithmIdentifier::SHA_1, builtins::CryptoAlgorithmIdentifier::SHA_224,
+    builtins::CryptoAlgorithmIdentifier::SHA_256, builtins::CryptoAlgorithmIdentifier::SHA_384,
+    builtins::CryptoAlgorithmIdentifier::SHA_512};
+std::vector<builtins::CryptoAlgorithmIdentifier> supportedGenerateKeyAlgorithms{
+    builtins::CryptoAlgorithmIdentifier::RSAES_PKCS1_v1_5,
+    builtins::CryptoAlgorithmIdentifier::RSASSA_PKCS1_v1_5,
+    builtins::CryptoAlgorithmIdentifier::RSA_PSS,
+    builtins::CryptoAlgorithmIdentifier::RSA_OAEP,
+    builtins::CryptoAlgorithmIdentifier::AES_CTR,
+    builtins::CryptoAlgorithmIdentifier::AES_CBC,
+    builtins::CryptoAlgorithmIdentifier::AES_GCM,
+    builtins::CryptoAlgorithmIdentifier::AES_CFB,
+    builtins::CryptoAlgorithmIdentifier::AES_KW,
+    builtins::CryptoAlgorithmIdentifier::HMAC,
+    builtins::CryptoAlgorithmIdentifier::ECDSA,
+    builtins::CryptoAlgorithmIdentifier::ECDH};
+std::vector<builtins::CryptoAlgorithmIdentifier> supportedDeriveBitsAlgorithms{
+    builtins::CryptoAlgorithmIdentifier::ECDH, builtins::CryptoAlgorithmIdentifier::HKDF,
+    builtins::CryptoAlgorithmIdentifier::PBKDF2};
+std::vector<builtins::CryptoAlgorithmIdentifier> supportedImportKeyAlgorithms{
+    builtins::CryptoAlgorithmIdentifier::RSAES_PKCS1_v1_5,
+    builtins::CryptoAlgorithmIdentifier::RSASSA_PKCS1_v1_5,
+    builtins::CryptoAlgorithmIdentifier::RSA_PSS,
+    builtins::CryptoAlgorithmIdentifier::RSA_OAEP,
+    builtins::CryptoAlgorithmIdentifier::AES_CTR,
+    builtins::CryptoAlgorithmIdentifier::AES_CBC,
+    builtins::CryptoAlgorithmIdentifier::AES_GCM,
+    builtins::CryptoAlgorithmIdentifier::AES_CFB,
+    builtins::CryptoAlgorithmIdentifier::AES_KW,
+    builtins::CryptoAlgorithmIdentifier::HMAC,
+    builtins::CryptoAlgorithmIdentifier::ECDSA,
+    builtins::CryptoAlgorithmIdentifier::ECDH,
+    builtins::CryptoAlgorithmIdentifier::HKDF,
+    builtins::CryptoAlgorithmIdentifier::PBKDF2};
+std::vector<builtins::CryptoAlgorithmIdentifier> supportedWrapKeyAlgorithms{
+    builtins::CryptoAlgorithmIdentifier::AES_KW};
+std::vector<builtins::CryptoAlgorithmIdentifier> supportedUnwrapKeyAlgorithms =
+    supportedWrapKeyAlgorithms;
+std::vector<builtins::CryptoAlgorithmIdentifier> supportedGetKeyLengthAlgorithms{
+    builtins::CryptoAlgorithmIdentifier::AES_CTR, builtins::CryptoAlgorithmIdentifier::AES_CBC,
+    builtins::CryptoAlgorithmIdentifier::AES_GCM, builtins::CryptoAlgorithmIdentifier::AES_CFB,
+    builtins::CryptoAlgorithmIdentifier::AES_KW,  builtins::CryptoAlgorithmIdentifier::HMAC,
+    builtins::CryptoAlgorithmIdentifier::HKDF,    builtins::CryptoAlgorithmIdentifier::PBKDF2};
+} // namespace
+namespace builtins {
```

Comment:
No need to change this, but it's totally fine to nest anonymous namespaces within another -- it would have been fine to leave the definition of the `builtins` namespace on line 25.